pennylane/qinfo/transforms.py

```diff
@@ -51,6 +51,8 @@ def circuit(x):
 
     .. seealso:: :func:`pennylane.density_matrix` and :func:`pennylane.math.reduce_dm`
     """
+    wire_map = {w: i for i, w in enumerate(qnode.device.wires)}
+    indices = [wire_map[w] for w in wires]
```

Comment:
I considered this option but wasn't sure if the new device API would have this `map_wires` method, since devices would no longer be responsible for keeping track of used wires. (Even `qnode.device.wires` would need to be changed to `qnode.tape.wires` once it's integrated). So I'd like to keep it as is.

Reply:
Fair enough :)